src/context/Note/context.js

```diff
@@ -52,8 +53,7 @@ const NoteProvider = ({ children }) => {
         {
           headers: { authorization: token },
         }
-      );
-      console.log("Updated Res", res);
+      );      
       setUserNotes(res.data.notes);
     } catch (error) {
       console.error("Update Notes Error", error);
```

Comment:
If you are using toast for messages, then this console.log can be moved to toast.

Reply:
Yeah will do that